vg/vg.go

```diff
@@ -15,24 +15,21 @@ import (
 // graphics.  The origin is in the bottom left corner.
 type Canvas interface {
 	// SetLineWidth sets the width of stroked paths.
-	// If the width is set to 0 then stroked lines are
-	// not drawn.
+	// If the width is not positive then stroked lines
+	// are not drawn.
 	//
 	// The initial line width is 1 point.
 	SetLineWidth(Length)
 
 	// SetLineDash sets the dash pattern for lines.
-	// The first argument is the pattern (units on,
```

Comment:
I guess that the part I was concerned with was the description of the pattern: units on first, units off second. But perhaps that's obvious?